build.gradle

```diff
@@ -32,7 +32,7 @@ task clean(type: Delete) {
   delete rootProject.buildDir
 }
 
-def BpkVersion = '0.2.1'
+def BpkVersion = '0.2.2'
```

Comment:
is the version bump done on purpose?

Reply:
This was done by the `release-it.js` task. Will reverse it for now.